packages/react-components/src/components/MessageThread.tsx

```diff
@@ -341,7 +334,7 @@ const memoizeAllMessages = memoizeFnAll(
                 onRenderMessageStatus ? (
                   onRenderMessageStatus({ status: statusToRender })
                 ) : (
-                  MessageStatusIndicator({ status: statusToRender })
+                  defaultStatusRenderer(statusToRender)
```

Comment:
Since this returns a JSX component. Can we Capitalize the first letter?
`DefaultStatusRenderer`
```suggestion
                  DefaultStatusRenderer(statusToRender)
```